pydantic/generics.py

```diff
@@ -119,6 +160,20 @@ def check_parameters_count(cls: Type[GenericModel], parameters: Tuple[Any, ...])
         raise TypeError(f'Too {description} parameters for {cls.__name__}; actual {actual}, expected {expected}')
 
 
+def iter_contained_typevars(v: Any) -> Iterator[TypeVarType]:
+    if isinstance(v, cast(type, TypeVar)):  # mypy < 0.790 needs this cast
```

Comment:
we're now using `mypy` 0.790, so can this be removed?

Any docstring please.

Reply:
Better?  There's also a test specific to this function.  If you'd like I can add an example in the docstring, I'm unsure about the policy of docstring examples vs. tests (I don't think doctests are being run in ci if I'm not mistaken)